german/oktoberfest.md

```diff
@@ -0,0 +1,6 @@
+Sie besuchen Wiesn beim Oktoberfest. Sie sehen ein sch�nes Maedchen in einem gr�nen Dirndl. 
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.